backend/models/response_user.py

```diff
@@ -0,0 +1,16 @@
+from backend.models.user import User
+
+
+# This class should be used for sending users information to client.
+# We don't want to send sensitive information such as password or email.
+class ResponseUser:
```

Comment:
Is there a reason you need both this class and the User class? Can't you have one class and only send the relevant information? If you do need another class, can you inherit from the first one and just change the __repr__ function?

Reply:
we decided to use only one class instead of two. changes will appear soon